src-tauri/src/github/release_notes.rs

```diff
@@ -0,0 +1,48 @@
+use std::vec::Vec;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ReleaseInfo {
+    pub name: String,
+    pub published_at: String,
+    pub body: String
+}
+
+#[tauri::command]
+pub async fn get_northstar_release_notes() -> Result<Vec<ReleaseInfo>, String> {
+    println!("Fetching releases notes from GitHub API");
+
+    let url = "https://api.github.com/repos/R2Northstar/Northstar/releases";
+    let user_agent = "R2Northstar/Northstar";
```

Comment:
```suggestion
    let user_agent = "GeckoEidechse/FlightCore";
```

to match with https://github.com/GeckoEidechse/FlightCore/blob/1ae49de2b49a0be84b4a842e5a6a2938bcf1889c/src-tauri/src/lib.rs#L365

Reply:
Can we put this in one place, instead of having duplicates?

Reply:
We should yeah, I'm planning to splice out a bunch of things to global constants but that would be it's one PR. I'd suggest simply updating string for this PR and then making issue for global consts.

I'll also look into moving my GitHub related code into your module once this PR is merged.